bin/wdq-batch

```diff
@@ -137,17 +146,25 @@ job.add_condor_cmd('getenv', 'True')
 job.add_condor_cmd('accounting_group', args.condor_accounting_group)
 job.add_condor_cmd('accounting_group_user', args.condor_accounting_group_user)
 if args.universe != 'local':
-    job.add_condor_cmd('request_memory', 4096)
+    if 'wpipeline' in args.wpipeline:
+        job.add_condor_cmd('request_memory', 4096)
+    else:
+        job.add_condor_cmd('request_memory', 32768)
```

Comment:
I think we should condense this a little:

```python
if args.universe != 'local' and args.wpipeline.endswith('wpipeline):
    job.add_condor_cmd('request_memory', 4096)
elif args.universe != 'local':
    jobs.add_condor_cmd('request_memory', 32768)
```

Reply:
Good suggestion, I'll change this with the next commit.